src/reader/widgets/ReaderWidget.vue

```diff
@@ -4,14 +4,32 @@
       <div class="reader-container u-flex">
         <Paginator :urn="previous" direction="left" />
         <LoaderBall v-if="gqlLoading" />
-        <template v-else-if="imageMode">
-          <Reader :lines="lines" :textSize="textSize" :textWidth="textWidth" />
+        <div class="image-mode" :class="showImage" v-else-if="imageMode">
+          <ImageViewerToolbar :show="showImage" @show="onShowImage" />
```

Comment:
Not a blocker (and wasn't in https://github.com/scaife-viewer/explorehomer/pull/42/files#diff-7342f14685f94a5222b1cd6c709fb90cR100), but I think in the future we might want to establish "blocks" (really slots) for toolbars for these alternate reader modes.

Have captured within ![](https://github.trello.services/images/mini-trello-icon.png) [Modularity of readers / reader components](https://trello.com/c/aDn12rrP)

Reply:
I'm planning to extract a reusable toolbar between this and the map one